VideoLocker/src/main/java/org/edx/mobile/view/CourseVideoListActivity.java

```diff
@@ -122,90 +119,39 @@ protected void modeChanged(){
     }
 
     @Override
-    protected void updateDownloadProgress(final int progressPercent){
-
-        mHandler.postDelayed(new Runnable() {
-            @Override
-            public void run() {
-                if (progressPercent < 100) {
-                    downloadProgressBar.setVisibility(View.VISIBLE);
-                    mHandler.removeCallbacks(mHideRunnable);
-                    if (downloadIndicator.getAnimation() == null) {
-                        downloadIndicator.setVisibility(View.VISIBLE);
-                        Animation animation = AnimationUtils.loadAnimation(CourseVideoListActivity.this, R.anim.rotate);
-                        downloadIndicator.startAnimation(animation);
-                    }
-                } else { //progressPercent == 100
-                    downloadIndicator.clearAnimation();
-                    downloadIndicator.setVisibility(View.INVISIBLE);
-                    mHandler.postDelayed(mHideRunnable, getResources().getInteger(R.integer.message_delay));
-                }
-            }
-        }, 500);
-    }
-
-    @Override
-    protected void setVisibilityForDownloadProgressView(boolean show){
-         boolean visible = downloadProgressBar.getVisibility() == View.VISIBLE;
-        if (visible == show )
-            return; //do nothing
-
-        if ( show ){
-            //TODO - we pass a value less than 100 to indicate it is downloading.
-            updateDownloadProgress(0);
-        } else {
-            updateDownloadProgress(100);
-        }
-    }
+    protected void updateDownloadProgress(final int progressPercent){}
 
     @Override
     public void onDownloadSuccess(Long result) {
-        try {
-            updateListUI();
-        }catch(Exception e){
-            logger.error(e);
-        }
+        updateListUI();
     }
 
     @Override
     public void onDownloadFailure() {
+        updateListUI();
     }
 
     @Override
-    public void showProgressDialog(int numDownloads) {
-        setVisibilityForDownloadProgressView(true);
-    }
+    public void showProgressDialog(int numDownloads) {}
```

Comment:
I think we can remove this callback method from the base interface, as it doesn't seem to be in use now.